src/Microsoft.Health.Extensions.DependencyInjection/Microsoft.Health.Extensions.DependencyInjection.csproj

```diff
@@ -2,19 +2,12 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Does this conflict with having netcoreapp2.2 in the directory props?

Reply:
It overrides it

Reply:
I had to remove this from the directory props file. VS wasn't happy with it missing from the csproj files.